GitExtensionsVSIX/GitExtensionsVSIX.csproj

```diff
@@ -1,19 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
I'm curious: why did you have to explicitly import the SDK here? Why not use the `Sdk` attribute on the `Project` element?

Reply:
That project kept me awake for few hours.
VSIX projects aren't easily convertible to SDK-style (at least I couldn't find an easy way to do it). I found [one discussion](https://github.com/Microsoft/VSProjectSystem/issues/257) about it with [an example](https://github.com/CosmosOS/XSharp/blob/a8c23c716b5995b40771535b6c9620a941aba0e9/source/XSharp.ProjectSystem/XSharp.ProjectSystem.csproj) that helped me to get it building. 
I couldn't also find how to use `EnableDefaultCompileItems` and `EnableDefaultItems` properties.

If you can make it better, please go ahead. I'm keen to learn how.